core/drivers/gic.c

```diff
@@ -389,22 +389,40 @@ static void gic_it_set_pending(struct gic_data *gd, size_t it)
 	io_write32(gd->gicd_base + GICD_ISPENDR(idx), mask);
 }
 
-static void gic_it_raise_sgi(struct gic_data *gd, size_t it,
-		uint8_t cpu_mask, uint8_t group)
+static void gic_it_raise_sgi(struct gic_data *gd __maybe_unused, size_t it,
+			     uint8_t cpu_mask, uint8_t group)
 {
 	uint32_t mask_id = it & 0xf;
-	uint32_t mask_group = group & 0x1;
 	uint32_t mask_cpu = cpu_mask & 0xff;
+#if defined(CFG_ARM_GICV3)
+	/* Only support sending SGI to the cores in the same cluster now */
+	uint64_t mpidr = read_mpidr();
+	uint64_t mask_aff1 = (mpidr & MPIDR_AFF1_MASK) >> MPIDR_AFF1_SHIFT;
+	uint64_t mask_aff2 = (mpidr & MPIDR_AFF2_MASK) >> MPIDR_AFF2_SHIFT;
```

Comment:
Aff3 missing?

Reply:
I'll add it in the updated patch also, thanks!